hack/generate-db-migrations.sh

```diff
@@ -37,8 +37,14 @@ done
 
 . tests/postgres.env
 
-#for MAC docker run -t -v `pwd`:/bayesian \
-docker run -t -v "$(readlink -f "${THISDIR}/.."):/f8a_worker:z" \
+if [ "$(uname)" == "Darwin" ]; then
+    #for MAC docker
+    vol_mount="`pwd`:/f8a_worker:z"
+else
+    vol_mount="$(readlink -f "${THISDIR}/.."):/f8a_worker:z"
+fi
+
+docker run -t -v $vol_mount \
```

Comment:
I _think_ this will break for volume paths containing spaces.

Reply:
I.e. you want to retain the `" "`

Reply:
Nice catch, Thanks

Reply:
I meant doing something like this:
`docker run -t -v "$vol_mount"`

:)